src/utils/ChannelFetch.ts

```diff
@@ -0,0 +1,48 @@
+import type { Channel } from "@storybook/channels";
+
+export const FETCH_ABORTED = "ChannelFetch/aborted";
+export const FETCH_REQUEST = "ChannelFetch/request";
+export const FETCH_RESPONSE = "ChannelFetch/response";
```

Comment:
Please namespace these events, as-is are a bit too generic.

A common practice is to prefix them with the addon's name/id.

Reply:
I imagine this would be a generic Storybook utility, not something specific to this addon. I suppose I could prefix them until then.

Reply:
Generic storybook APIs should not come from an addon.

This gives the impression it's a core feature, but then it's gone, if a users disabled/removed the addon.

Reply:
Ok, I namespaced them and moved to the addon constants file.